src/util/misc.rs

```diff
@@ -10,6 +10,52 @@ use std::time::{SystemTime, UNIX_EPOCH};
 use crate::args::Args;
 use super::db::delete;
 
+pub fn cleanup_orphaned_files(pastas: &Vec<Pasta>, args: &Args) {
```

Comment:
To me it makes more sense to have both cleanup_orphaned_files & remove_expired in cleanup.rs. What do you think?